src/components/molecules/card/card.scss

```
@@ -0,0 +1,46 @@
+@use "../../variables" as *;
+.card {
+  border: none;
+  & > .img-wrapper {
+    overflow: hidden;
+    & > img {
+      width: 100%;
+      transition: 0.5s ease;
+    }
+    @media screen and (min-width: $breakpoint-md) {
+      &:hover > img {
+        transform: scale(1.2);
+      }
+    }
```

これでも書けるので参考までに！
前回コメントしなかったので今回はこのままで大丈夫です。
```suggestion
    & > img {
      width: 100%;
      transition: 0.5s ease;      
      @media screen and (min-width: $breakpoint-md) {
        &:hover {
         transform: scale(1.2);
         }
        }
      }
    }
```